frontend/public/components/dashboard/dashboards-page/cluster-dashboard/utilization-card.tsx

```diff
@@ -168,42 +168,67 @@ export const PrometheusUtilizationItem = withDashboardResources<PrometheusUtiliz
     utilizationQuery,
     totalQuery,
     duration,
+    adjustDuration = (start: number) => start,
     title,
     TopConsumerPopover,
     humanizeValue,
     byteDataType,
     setTimestamps,
+    isDisabled = false,
   }) => {
+    let stats = [];
```

Comment:
Im not a fan of so many `let` variables, seems a bit messy to me. Cant we just keep it as it was and set error for `UtilizationItem` if `isDisabled === true`  ?


```
return (
      <UtilizationItem
        title={title}
        data={stats}
        error={isDisabled || utilizationError || totalError}
        isLoading={!utilization || (totalQuery && !total)}
        isLoading={isLoading}
        humanizeValue={humanizeValue}
        byteDataType={byteDataType}
        query={utilizationQuery}
        max={max.length ? max[0].y : null}
        max={max && max.length ? max[0].y : null}
        TopConsumerPopover={TopConsumerPopover}
      />
    );
```

Reply:
Actually that was my first implementation. But with the later one (means the recent one), various calls are completely skipped for `isDisabled == true`. Wdyt?

Reply:
Looking at the skipped calls, they are very simple so calling them wont have any real performance impact and we will have a code which is easier to read. So Im still partial to proposed solution (removing the let variables and just adding isDisabled to error field)